migrate/loader.go

```diff
@@ -0,0 +1,162 @@
+package migrate
+
+import (
+	"fmt"
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+	"os"
+	"strings"
+)
+
+var loaderCommand = &cobra.Command{
+	Use:   "loader [version|latest|recommended]",
+	Short: "Migrate your modloader version to a newer version.",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		modpack, err := core.LoadPack()
+		if err != nil {
+			// Check if it's a no such file or directory error
+			if os.IsNotExist(err) {
+				fmt.Println("No pack.toml file found, run 'packwiz init' to create one!")
+				os.Exit(1)
+			}
+			fmt.Printf("Error loading pack: %s\n", err)
+			os.Exit(1)
+		}
+		// Get our current loader, would use the modpack.GetLoaders function
+		// but, it supplements Quilt with Fabric, which isn't needed for this
+		var currentLoaders []string
+		// Add all the keys from modpack.Versions
```

Comment:
Probably a good idea to filter this by the supported loaders in `core.ModLoaders`; old versions of packwiz [used a yarn field](https://github.com/Fibercraft/Temporary-Modpack/blob/f33038291c48e7978b47b2a4453f11b768fae9e4/pack.toml#L11) and unsup adds [it's own version field for format extensions](https://git.sleeping.town/unascribed-tools/unsup/wiki/Packwiz-extensions).